rocAL/rocAL/include/meta_data/meta_node_crop_mirror_normalize.h

```diff
@@ -32,7 +32,7 @@ class CropMirrorNormalizeMetaNode:public MetaNode
 {
     public:
         CropMirrorNormalizeMetaNode() {};
-        void update_parameters(MetaDataBatch* input_meta_data)override;
+        void update_parameters(MetaDataBatch* input_meta_data, bool segmentation)override;
```

Comment:
Can we add the segmentation flag as part of MetaDatabatch / info?
That way we need not pass these flags. 
Also how do we handle the keypoints when it comes to updation?
We need not add more flags and just add these as part of struct.

Reply:
Done